test/integration/programs/sidebar/action-sidebar.js

```diff
@@ -324,6 +329,12 @@ context('program action sidebar', function() {
         fx.data[2].attributes.published_at = testTs();
         fx.data[3].attributes.name = 'E Form';
         fx.data[3].attributes.published_at = testTs();
+
+        return fx;
+      })
+      .routeWorkspaces(fx => {
+        fx.data[0].relationships.forms = { data: _.first(getRelationship(fxForms, 'forms'), 6) };
+
```

Comment:
Without this, this shows in the UI of the test:

![Screenshot 2023-03-27 at 2 25 02 PM](https://user-images.githubusercontent.com/35355575/228046977-66b6aa95-cc1b-407e-b867-136ba1850a41.png)

The test will still pass without this though.

Reply:
could just be `fx.data[0].relationships.forms.data = _.first(fx.data[0].relationships.forms.data), 6);`

alternatively use `fxForms` here and in `routeForms` and do the `_.first` prior to the `getRelationship`/ `getResource`

Reply:
I updated it to the first option: `fx.data[0].relationships.forms.data = _.first(fx.data[0].relationships.forms.data), 6);`. Seems the simplest. And I like not having to import anything.